addons/payment_razorpay/const.py

```diff
@@ -145,4 +145,9 @@
     'payment.failed',
     'refund.failed',
     'refund.processed',
+    'account.app.authorization_revoked',
 ]
+
+# TODO: change IAP url to the correct one (right now it's a test url)
+OAUTH_URL = "https://razorpay.api.odoo.com"
+OAUTH_TEST_URL = "https://razorpay.test.odoo.com"
```

Comment:
In my opinion, this should be added in the system parameters to make changes easily